server/src/main/java/io/deephaven/server/hierarchicaltable/HierarchicalTableServiceGrpcImpl.java

```diff
@@ -238,7 +238,7 @@ public void apply(
                                         makeWhereFilters(finishedConditions, nodeDefinition);
                                 RollupTableImpl.initializeAndValidateFilters(
                                         rollupTable.getSource(),
-                                        rollupTable.getGroupByColumns(),
+                                        rollupTable.getAggregations(),
```

Comment:
Somewhere in here is where we will need to do at least part of https://deephaven.atlassian.net/browse/DH-19279, which is separate ticket as you are not changing the gRPC functionality here.